src/services/mocksdk.ts

```diff
@@ -0,0 +1,27 @@
+import { generateRandomSocialGraph } from "../test/testGraphs";
+import { generateProfile } from "../test/testProfiles";
+import { Graph, HexString, Profile } from "../utilities/types";
+
```

Comment:
check out this story to see what I want to do.  what blockers do we have on implementing this?